scripts/userstyles.yml

```diff
@@ -960,4 +970,4 @@ userstyles:
           answer: "It's available in case you have an OLED display. If you have one, you might want to enable this."
     current-maintainers: [*isabelroses, *uncenter]
     past-maintainers: [*elkrien]
-# yaml-language-server: $schema=userstyles.schema.json
+# yaml-language-server: $schema=userstyles.schema.json
```

Comment:
Please keep our trailing newline! This repository uses https://editorconfig.org/, so make sure your editor has the appropriate extension/tooling installed to make it work.